components/dashboard/src/AppNotifications.tsx

```diff
@@ -113,10 +112,10 @@ export function AppNotifications() {
                     const [subscriptionId, invalidBillingAddress] = await Promise.all([
                         getGitpodService().server.findStripeSubscriptionId(attributionId),
                         getGitpodService().server.isCustomerBillingAddressInvalid(attributionId),
-                        getGitpodService().server.getStripePortalUrl(attributionId).then(setStripePortalUrl),
```

Comment:
Keep the `getStripePortalUrl`  call here so it's awaited at the same time just remove the `then`

Reply:
but, then will it update the value to `stripePortalUrl`?

Reply:
resolved in [1cc320d](https://github.com/gitpod-io/gitpod/pull/19431/commits/1cc320df184a81ed51efa012a802ec15ec92de42)